atom/browser/api/atom_api_update_client.cc

```diff
@@ -0,0 +1,172 @@
+// Copyright 2016 The Brave Authors. All rights reserved.
```

Comment:
can we move this to brave/browser in a follow-up? Trying to keep non-upstreamable stuff separate so we can split out the repos